src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -267,7 +267,7 @@
           (cond-> {:dispatch [:navigate-back]}
             (= method constants/web3-send-transaction)
             (assoc :dispatch-later [{:ms 400 :dispatch [:navigate-to-modal :wallet-transaction-sent-modal]}]))
-          {:dispatch [:navigation-replace :wallet-transaction-sent]}))))))
```

Comment:
I don't understand why we have to break otherwise atomic logic in `::transaction-completed` into two distinct events instead of just having the handler for `:execute-stored-command` defined as public (model) function and composing it inside `::transaction-completed` event.

Reply:
My point here is pretty simple, i want to touch as less code as possible. I don't know why it was written this way in the first place.

Reply:
Sure, but you actually added another unnecessary event, so it's slippery slope.

Reply:
one of the reasons might be that that there are some conditions and devs found it not obvious how to merge effects, so it was just safer to dispatch another event, idk